src/Jackalope/NodeType/NodeProcessor.php

```diff
@@ -81,9 +109,11 @@ public function __construct(
     /**
      * Process the given node.
      *
+     * TODO: Either we should have this method return the additional operations, or call getAdditionalOperations
```

Comment:
why does this method not return the additional operations directly? and it seems in the integration, we missed to handle this.